src/script/sconsign.py

```diff
@@ -270,19 +277,20 @@ def map_timestamp(entry, name):
     else:
         return str(timestamp)
 
-def map_bkids(entry, name):
+def map_bkids(entry, _):
     try:
         bkids = entry.bsources + entry.bdepends + entry.bimplicit
         bkidsigs = entry.bsourcesigs + entry.bdependsigs + entry.bimplicitsigs
     except AttributeError:
         return None
     result = []
-    for i in range(len(bkids)):
-        result.append(nodeinfo_string(bkids[i], bkidsigs[i], "        "))
+    for bkid, bkidsig in zip(bkids, bkidsigs):
```

Comment:
This could be list comprehension.. (which is faster than for loop.. ;)

Reply:
done... though I don't think this is a performance critial place, as it's outputting a bunch of strings to stdout and won't ever do anything with results beyond that....